packages/terra-site/tests/jest/theming-plugin-spec.test.jsx

```diff
@@ -54,9 +54,8 @@ describe('theming-plugin', () => {
         },
       };
       rootNode.source = { input: { file: 'TestingComponent.scss' } };
-
       expect(testingFunction(rootNode)).toEqual(JSON.stringify(expectedHash));
-      expect(outputtedFileName).toEqual('themeable-variables.json');
+      expect(outputtedFileName.slice(outputtedFileName.length - 55)).toEqual('terra-core/packages/terra-site/themeable-variables.json');
```

Comment:
What does -55 do?

Reply:
![](https://media.giphy.com/media/YnlXLCYDeeE3m/giphy.gif)

Reply:
Wanted to grab the end of the file / path, or the last 55 characters. Using this to check that its within the `terra-site` folder. This slice returns `terra-core/packages/terra-site/themeable-variables.json`. Depending on where this code is run, your machine, my machine, travis, the outputtedFileName's file path is different. Slicing the string to just the path thats consistent gives us the info we need when run in various environments. Also, its a nod to not driving slow.